manifest.json

```diff
@@ -1865,6 +1865,27 @@
       "path": "plugins/minimap.lua",
       "version": "0.2"
     },
+    {
+      {
+      "id": "modal",
+      "description": "Fully customizable modal editing for Lite XL",
+      "mod_version": "3",
+      "name": "modal",
+      "remote": "https://codeberg.org/Mandarancio/lite-modal:a770b1673b8cb2c5f80176c0a3529bdafe3e482a",
+      "type": "plugin",
+      "version": "0.2",
+      "dependencies": {
+        "lsp": {
+          "optional": true
+        },
+        "lintplus": {
+          "optional": true
+        },
+        "tab_switcher": {
+          "optional": true
+        }
+      }
+    },
```

Comment:
This is a level too much of object nesting.

Also I think you can remove the dependencies, as they'll get picked up from the manifest in your repo.